.github/workflows/ci.yaml

```
@@ -0,0 +1,29 @@
+name: Python Tests
+
+on:
+  push:
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        python-version: [3.9, 3.10, 3.11, 3.12, 3.13]
```

```suggestion
        python-version: [3.9, '3.10', 3.11, 3.12, 3.13]
```
yaml parses unquoted value as a number and truncates it. https://github.com/actions/setup-python/issues/160#issuecomment-724485470